docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -0,0 +1,207 @@
+//[id="proc_importing_code_{context}"]
+
+= Importing and configuring code
+
+[role="_abstract"]
+After you click **Create an application**, the **Grab some code** page opens. From here, choose one of the following source code options to begin building your application:
+
+* **Bring in your own code**
+* **Start with a sample**
+
+== Using a code sample
+
+. Optional: In the code sample **Search** field, begin typing the name of the code you want to work with. If {ProductName} has code samples in that language stack, the code sample results filter accordingly.
+. Optional: If you see a language stack or framework that you want to work with, click either its **Open Git repository** or **Import sample** link.    
+* If you click **Open Git repository**, a new tab opens to the GitHub repository for the code framework you chose.
+* If you click **Import sample**, {ProductName} creates and deploys a new application with the sample code you chose. Check out the open **Overview** tab to monitor your continuous integration/continuous deployment (CI/CD) activity.
```

Comment:
It's a minor detail, but you could mention here that applications created from samples will be automatically named.

Reply:
added, @mattreid thank. can users change it?